daemon/daemon.go

```diff
@@ -127,14 +127,6 @@ func (d *Daemon) RemoveProxyRedirect(e *endpoint.Endpoint, l4 *policy.L4Filter)
 	return d.l7Proxy.RemoveRedirect(id)
 }
 
-func (d *Daemon) WriteEndpoint(e *endpoint.Endpoint) error {
-	if err := d.conf.LXCMap.WriteEndpoint(e); err != nil {
```

Comment:
lxcmap instance is now maintained in the package itself. There is always only a single instance.